src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/note/NoteEntity.kt

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.note
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import org.hibernate.Hibernate
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.UpdateTimestamp
+import org.hibernate.annotations.UuidGenerator
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.LastModifiedBy
+import java.time.Instant
+import java.util.UUID
+
+@Table(name = "note")
+@Entity
+class NoteEntity(
+  @Id
+  @GeneratedValue
+  @UuidGenerator
+  var id: UUID? = null,
+
+  @Column
+  var prisonNumber: String? = null,
+
+  @Column(updatable = false)
+  @field:NotNull
+  var reference: UUID? = null,
+
+  @Column
+  @field:NotNull
+  var content: String? = null,
+
+  @Column
+  var noteType: String? = null,
```

Comment:
We should make these type fields enums - see what we've done in other entity classes